src/screens/HomeScreen.js

```diff
@@ -230,27 +243,36 @@ export const HomeScreen = ({ navigation }) => (
                   message: shareMessage(eventRecord, 'eventRecord')
                 }
               },
+              bottomDivider: index === data.eventRecords.length - 1 ? false : true,
               __typename: eventRecord.__typename
             }));
 
           if (!eventRecords.length) return null;
 
-          // add index element, if not already present - check for the element with id: '-1'
-          eventRecords.map((eventRecord) => eventRecord.id).indexOf('-1') < 0 &&
-            eventRecords.push({
-              id: '-1',
-              title: 'Alle Veranstaltungen anzeigen',
-              routeName: 'Index',
-              params: {
-                title: 'Veranstaltungen',
-                query: 'eventRecords',
-                queryVariables: {},
-                rootRouteName: 'EventRecords'
-              },
-              bottomDivider: false
-            });
+          return (
+            <View>
+              <TextList navigation={navigation} data={eventRecords} />
 
-          return <TextList navigation={navigation} data={eventRecords} alternativeLayout />;
+              {!!eventRecords && (
```

Comment:
since we are checking for the `length` of the `eventRecords` object in line 250, we do not need this check here (again). it is safe to render `<Wrapper><Button...` directly.